src/main/java/net/masterthought/cucumber/CucumberReportGeneratorMojo.java

```diff
@@ -86,6 +95,11 @@ public void execute() throws MojoExecutionException {
             Configuration configuration = new Configuration(outputDirectory, projectName);
             configuration.setBuildNumber(buildNumber);
             configuration.setParallelTesting(parallelTesting);
+            if (classifications != null) {
```

Comment:
I thought when it is empty then the "for" loop will just be not executed (.entrySet() being empty, i.e. loop will be skipped immediately)?